See the following issue: https://github.com/spillerrec/qt-flif-plugin/issues/5
This patch is not a real fix.

--- a/src/Flif++.hpp
+++ b/src/Flif++.hpp
@@ -78,8 +78,8 @@
 		void setResize( uint32_t width, uint32_t height )
 			{ flif_decoder_set_resize( d, width, height ); }
 		
-		void setCallback( uint32_t (*callback)(int32_t,int64_t) )
-			{ flif_decoder_set_callback( d, callback ); }
+//		void setCallback( uint32_t (*callback)(int32_t,int64_t) )
+//			{ flif_decoder_set_callback( d, callback ); }
 		
 		void setFirstCallbackQuality( int32_t quality )
 			{ flif_decoder_set_first_callback_quality( d, quality ); }
